Dockerfile

```diff
@@ -27,7 +29,18 @@ WORKDIR /gocode/src/github.com/replit/prybar
 
 RUN which node
 
-RUN make prybar-python2 prybar-python3 prybar-ruby prybar-lua prybar-spidermonkey prybar-nodejs
+# OCaml / Reason stuff
+RUN add-apt-repository ppa:avsm/ppa && \
+  apt-get update && \
+  apt-get install opam -y && \
+  opam init -y --disable-sandboxing && \
+  eval `opam env` && \
+  echo "eval \`opam env\`" >> ~/.bashrc && \
+  opam update && \
+  opam switch create 4.06.1 && \
+  opam install reason.3.4.0 -y
```

Comment:
This command hangs for me when I try to docker build on OSX.

This bit is kinda scary as well

```
   Do you want opam to modify ~/.profile? [N/y/f]
   (default is 'no', use 'f' to choose a different file) A hook can be added to opam's init scripts to ensure that the shell remains in sync with the opam environment when they are loaded. Set that up? [y/N] y
```